lib/rpc/rpc.dart

```diff
@@ -30,8 +30,11 @@ class RPCLive implements RPC {
   }
 
   @override
-  Future<double> getBalance() async {
-    return await _client.call('getbalance') as double;
+  Future<(double, double)> getBalance() async {
+    final confirmed = await _client.call('getbalance') as double;
+    final unconfirmed = await _client.call('getunconfirmedbalance') as double;
+
+    return (confirmed, unconfirmed);
```

Comment:
what do we want to do here? Is this okay? Should we create a custom named class instead?

Reply:
Seems good to me